arch/Alpha/AlphaMapping.c

```diff
@@ -0,0 +1,117 @@
+#ifdef CAPSTONE_HAS_ALPHA
+
+#include <stdio.h> // debug
+#include <string.h>
+
+#include "../../utils.h"
+
+#include "AlphaMapping.h"
+#include "../../Mapping.h"
+
+#define GET_INSTRINFO_ENUM
+
+#include "AlphaGenInstrInfo.inc"
+
+static insn_map insns[] = {
+#include "AlphaGenCSMappingInsn.inc"
+};
+
+// unsigned int Alpha_map_insn_id(cs_struct *h, unsigned int id)
```

Comment:
Don't forget to checkout which of those mapping functions are already implemented in `Mapping.c`.